src/util/isObjectEmpty.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Determines whether the object is empty by checking whether the length
+ * of key for that object is false
```

Comment:
1. describing how the function works internally is not required - ie: `the length of keys` is an implementation detail and very wordy, `props` describes it just as well and much shorter, please use minimum of descriptive words

Reply:
2. rather use: `Checks if a value has any props`